core/src/test/scala/org/broadinstitute/dsde/rawls/mock/RemoteServicesMockServer.scala

```diff
@@ -551,5 +551,25 @@ class RemoteServicesMockServer(port:Int) extends RawlsTestUtils {
       )
   }
 
+  mockServer.when(
```

Comment:
If I refactor ever piece of bad code I see, I'll never end up checking this in :)